.github/workflows/ci.yml

```diff
@@ -14,7 +14,3 @@ jobs:
         run: npm run lint
       - name: Unit Tests
         run: npm run test:unit
-      - name: Coveralls
```

Comment:
Maintaining code coverage would have been kind of rough here, since nyc/istanbul appears to require a babel build step in order to work properly when covering ES6 modules. This seems like a hefty chunk of stuff to add to `devDependencies` considering what we gain from the coverage tools.